### PR TITLE
Category selector: prevent empty category name specified

### LINKED ISSUE #2148: Categories - Add: Prevent creating blank tables
Currently, use can accidentally create blank categories with no name. Please fix this. 

image

### PR DESCRIPTION
**Pull request by SteRiccio**
fixes #2148

### CODE REVIEW

File: webapp/components/hooks/useNotifyError.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { useNotification } from './useNotification'
+
+export const useNotifyError = () => useNotification({ severity: 'error' })
```
Comment (ramirobg94) [inline webapp/components/hooks/useNotifyError.js:3]: maybe this hooks can be a subtype of useNotification and can be added into 

useNotification/
...index.js
...useNotificationError 

if not the components/hooks could grow without any sense

File: webapp/components/survey/CategorySelector/CategorySelector.js [hunk 2]
```diff
@@ -4,20 +4,22 @@ import React, { useState, useEffect, useCallback } from 'react'
...
 
+import { useNotifyWarning } from '@webapp/components/hooks'
 import Dropdown from '@webapp/components/form/Dropdown'
 import PanelRight from '@webapp/components/PanelRight'
 import CategoryList from '@webapp/components/survey/CategoryList'
 import CategoryDetails from '@webapp/components/survey/CategoryDetails'
 import ButtonMetaItemAdd, { metaItemTypes } from '@webapp/components/survey/ButtonMetaItemAdd'
 
-const CategorySelector = (props) => {
+const CategorySelector = function (props) {
```
Comment (ramirobg94) [inline webapp/components/survey/CategorySelector/CategorySelector.js:22]: why this change?

File: webapp/components/hooks/useNotifyError.js (hunk 1 above)
Comment (SteRiccio) [reply webapp/components/hooks/useNotifyError.js:3]: uhm, I think I've understood!

File: webapp/components/survey/CategorySelector/CategorySelector.js (hunk 2 above)
Comment (SteRiccio) [reply webapp/components/survey/CategorySelector/CategorySelector.js:22]: eheheh I've thought the same thing... it's eslint that fixed it, I think it's because it's exported as default...

File: test/e2e/mock/user.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export const user = {
-  email: 'test@arena.com',
```
Comment (SteRiccio) [inline test/e2e/mock/user.js:2]: arena.com was really weird...

File: webapp/components/survey/CategorySelector/CategorySelector.js [hunk 4]
```diff
@@ -4,20 +4,22 @@ import React, { useState, useEffect, useCallback } from 'react'
...
 
+import { useNotifyWarning } from '@webapp/components/hooks'
 import Dropdown from '@webapp/components/form/Dropdown'
 import PanelRight from '@webapp/components/PanelRight'
 import CategoryList from '@webapp/components/survey/CategoryList'
 import CategoryDetails from '@webapp/components/survey/CategoryDetails'
 import ButtonMetaItemAdd, { metaItemTypes } from '@webapp/components/survey/ButtonMetaItemAdd'
 
-const CategorySelector = (props) => {
+export const CategorySelector = (props) => {
```
Comment (SteRiccio) [inline webapp/components/survey/CategorySelector/CategorySelector.js:22]: this is just a trying to avoid the eslint func-names rule to pass, but it doesn't work too...

**Review by ramirobg94 — APPROVED** [review]
